src/main/java/TaskManager.java

```diff
@@ -0,0 +1,215 @@
+import Exceptions.InvalidDeadlineFormatException;
+import Exceptions.InvalidEventFormatException;
+import Exceptions.InvalidTodoFormatException;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class TaskManager {
+    // private static final String FILE_PATH = "./data/tasks.txt";
+    private static final String FILE_PATH = "." + File.separator + "data" + File.separator + "tasks.txt";
+
+    public static final int DEADLINE_BEGIN_INDEX = 8;
+    public static final int EVENT_BEGIN_INDEX = 5;
+    public static final int TODO_BEGIN_INDEX = 4;
+    public static final int EVENT_MAX_PARTS = 3;
+    public static final int DEADLINE_MAX_PARTS = 2;
+    public static final int INDEX_OFFSET = 1;
+    public static final int PART_0 = 0;
+    public static final int PART_1 = 1;
+    public static final int PART_2 = 2;
+    public static final int START_INDEX = 0;
+    private ArrayList<Task> taskList = new ArrayList<>();
+    private int index = 0;
+    UserInterface userInterface = new UserInterface();
+
+    public void addTask(String taskDescription) {
+        String taskType = taskDescription.split(" ")[START_INDEX];
+        try {
+            switch (taskType) {
+                case "deadline":
```

Comment:
case here would have the same indentation level as switch according to coding standards